### ISSUE TITLE
Allow creation of new content from the ObjectBrowser popup sidebar

### ISSUE THREAD

**Issue by tiberiuichim** [plone/volto#1905]
Feasible?

**Comment by tisto** [comment]
@tiberiuichim my questions: what kind of content do you have in mind? Why is the normal way of adding stuff not sufficient?

**Comment by tiberiuichim** [comment]
Actually we always get this request, even with regular Plone.

So, using the ObjectBrowser as a "related items/internal link" browser, the editor searches for the content to link but realises that the content doesn't exist, it needs to be created. So now he has to abandon the current edit form to create the new content, then come back and resume/finish the edit.

**Comment by tisto** [comment]
Interesting. I never heard this kind of request from my clients. Clients told us they would prefer a more "tree-like" representation of the sidebar but that was it...

**Comment by tiberiuichim** [comment]
With regards to the creation of new objects in the ObjectBrowser, that's my personal wish, I've had to use it "in anger" in the last days and these are my ideas.

But for the "related items" widget from Plone, we're using it in another project where it's being used as an "object lookup" and I got the wish/request from the client to be able to create that content directly from the widget. Or at least a way to trigger this process.

In any case, this is of course achievable for a more advanced user, just open in a new tab, etc. I'll close it for now and think about it more.
